packages/components/src/color-palette/style.scss

```diff
@@ -25,3 +25,10 @@
 		padding: 0;
 	}
 }
+
+@include break-medium() {
+	.components-dropdown__content.components-color-palette__custom-color-dropdown-content.is-rendered-in-sidebar.is-from-top .components-popover__content {
```

Comment:
This is really coupled with internal class names of other components. Do you think we could avoid them or extract them somehow?

Reply:
Here we are changing the position of a popover, I'm not sure how we can avoid relying on the classes the component defines. And unfortunately, this is not the first time we do it, searching for ".components-popover__content" on the codebase we find lots of matches used on each place we change something related to the popover, in some cases even outside the components package while on this case we are inside the components package.
It is not an ideal solution but it seems to be consistent with what we are doing on similar cases.